accessibility_monitoring_platform/apps/audits/tests/test_models.py

```diff
@@ -1497,3 +1502,17 @@ def test_statement_page_str():
     statement_page.url = "url"
 
     assert str(statement_page) == "url"
+
+
+@pytest.mark.django_db
+def test_latest_statement_link_found():
+    """
+    Test that the latest statement link is returned even when
+    it is not on the latest statement page.
+    """
+    case: Case = Case.objects.create()
+    audit: Audit = Audit.objects.create(case=case)
+    StatementPage.objects.create(audit=audit, url=STATEMENT_LINK)
+    StatementPage.objects.create(audit=audit)
+
+    assert audit.latest_statement_link == STATEMENT_LINK
```

Comment:
Is there a test for when a statement link isn't found and what the expected behaviour is?

Reply:
Or if there is just backup link but no URL to the statement?

Reply:
I've added a test that `None` is returned when no link to the statement page has been entered.

As well as one that the *View latest accessibility statement* link in *Frequently used links* is not a link when none has been entered. f0f54341d9d44b15eb8e2b4ef250a098cb3e1295